src/Model/collections.ts

```diff
@@ -124,6 +133,10 @@ Model.prototype._getDeepCopy = function(segments) {
   return util.deepCopy(value);
 };
 
+Model.prototype.getValues = function<S>(subpath?: Path) {
+  return this.filter(subpath, null).get();
```

Comment:
Would be better to write out the logic explicitly, instead of re-using the confusing pattern internally. I'm thinking something like this:
```suggestion
  const value = this.get(subpath);
  if (value == null) {
    return [];
  }
  if (typeof value !== 'object') {
    throw new Error(`Found non-object type for getValues('${this.path(subpath)}')`);
  }
  return Object.values(value);
```